test/spec.index.js

```diff
@@ -65,6 +65,18 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('looks up default field label if nothing is set', function () {
```

Comment:
I think we should also add a test for custom label text and add tests for hint text for input text and radio groups.

I know I'm being pedantic, so happy to add these if you want.